src/Client.php

```diff
@@ -85,7 +85,7 @@ public function captureMessage(string $message, ?Severity $level = null, ?Scope
             'level' => $level,
         ];
 
-        $event = $this->prepareEvent($payload, $scope, $this->options->shouldAttachStacktrace());
+        $event = $this->prepareEvent($payload, $scope);
```

Comment:
Instead of calling `prepareEvent`, we can now simply do `return captureEvent`

Reply:
fixed